package.json

```diff
@@ -73,10 +73,11 @@
     "@azure/storage-blob": "^12.5.0",
     "@azure/storage-queue": "^12.4.0",
     "@lifeomic/attempt": "^3.0.0",
-    "@microsoft/microsoft-graph-client": "^2.0.0",
+    "@microsoft/microsoft-graph-client": "^3.0.2",
```

Comment:
@jleodany 

How come there was a major version upgrade (which usually brings breaking changes) and there was no other changes related to this package?

https://github.com/microsoftgraph/msgraph-sdk-javascript/blob/dev/changelogs/v3-upgrade-guide.md#breaking-changes

There were breaking changes in version 3 according to that changelog.

Reply:
@labs-dlugo This is a first time commit so no issues on doing this.

@jleodany This has been tested already right?

Reply:
@sebassdc yes, this is how it is working right now